### PR TITLE
Update to-start-group invitation mailer

### PR DESCRIPTION
**Pull request by HSalmon**
screen shot 2016-08-02 at 10 46 34 am

### CODE REVIEW

**Comment by HSalmon** [comment]
@rdbartlett 

screen shot 2016-08-04 at 12 22 20 pm
screen shot 2016-08-04 at 12 10 00 pm
screen shot 2016-08-04 at 12 41 15 pm
screen shot 2016-08-04 at 1 03 53 pm

**Comment by rdbartlett** [comment]
Be still my beating heart ❤️

File: spec/mailers/previews/invite_people_mailer_preview.rb [hunk 1]
```diff
@@ -21,8 +21,8 @@ def to_join_group
     invitation = InvitationService.create_invite_to_join_group(
                     recipient_email: recipient_email,
                     inviter: responder,
-                    group: group)
-    message_body = "Pleaseeeeee join us, it will complete the phophecy and we'll become captain planet"
```
Comment (gdpelican) [inline spec/mailers/previews/invite_people_mailer_preview.rb]: RIP the Phophecy.

**Comment by gdpelican** [comment]
🍏

**Pull request merged**